### DISCUSSION TITLE
Problem configuring service on RP ZW

### DISCUSSION THREAD

**Discussion by jane-t** [mKeRix/room-assistant#648] [Q&A] [answered]
I have two Pi's configured for RA.
The third one is a Pi Zero.

If I log in and go to the room-assistant directory and type room-assistant it starts correctly and joins the cluster.
I have copied in the service file from https://www.room-assistant.io/guide/quickstart-pi.html#installing-room-assistant and can start the service, changing the restart to never to see the messages shows
`room-assistant.service - room-assistant service
   Loaded: loaded (/etc/systemd/system/room-assistant.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2021-04-13 11:18:37 BST; 10min ago
  Process: 872 ExecStart=/usr/bin/room-assistant (code=exited, status=203/EXEC)
 Main PID: 872 (code=exited, status=203/EXEC)

Apr 13 11:18:37 rapiz-1 systemd[1]: Started room-assistant service.
Apr 13 11:18:37 rapiz-1 systemd[872]: room-assistant.service: Failed to execute command: No such file or directory
Apr 13 11:18:37 rapiz-1 systemd[872]: room-assistant.service: Failed at step EXEC spawning /usr/bin/room-assistant: No such file or directory
Apr 13 11:18:37 rapiz-1 systemd[1]: room-assistant.service: Main process exited, code=exited, status=203/EXEC
Apr 13 11:18:37 rapiz-1 systemd[1]: room-assistant.service: Failed with result 'exit-code'.`

Trying to run 
 /usr/bin/room-assistant
also gives an error.

Can anyone suggest what I have missed

Thanks

**Comment by mKeRix** [comment, marked as answer]
The quickstart guide for the Pi Zero W contains a [correct example](https://www.room-assistant.io/guide/quickstart-pi-zero-w.html#making-sure-it-always-runs) for the service file on that platform. Most importantly the ExecStart path should be `/opt/nodejs/bin/room-assistant`. The Pi Zero is a bit of a special platform, which is why the path differs... I'm thinking about making the setup experience a bit better in the future.

You can also verify the correct path to use by running `which room-assistant` on the Pi.

**Reply by jane-t** [reply]
Thanks - I knew it was something obvious I had done wrong,  I just could not spot it.  Having both sets of instructions open did not help.

**Reply by whedrick** [reply]
Thanks for the command to verify the path.  In my case, the correct path is /usr/local/bin/room-assistant

**Reply by Electronlibre2012** [reply]
@mKeRix thanks for the info...waste a lot of time to search the good path lol
